zerver/tests/test_slack_message_conversion.py

```diff
@@ -113,6 +113,11 @@ def test_has_link(self) -> None:
         self.assertEqual(text, "http://journals.plos.org/plosone/article")
         self.assertEqual(has_link, True)
 
+        message = "<http://chat.zulip.org/asd/ads/?asd=12093|Chat Zulip Org>"
```

Comment:
Maybe `<https://chat.zulip.org/help/logging-in|Help logging in to CZO>` since that's an example link that's not completely invalid?

Reply:
Done :+1: